app/src/main/java/com/amaze/filemanager/filesystem/compressed/CompressedMimeParsing.java

```diff
@@ -0,0 +1,22 @@
+package com.amaze.filemanager.filesystem.compressed;
+
+/**
+ * Parse the MIME type of a compressed file
+ */
+public final class CompressedMimeParsing {
+    public static boolean isFileCompressed(String mimeType) {
+        if(mimeType.startsWith("resource")) {
+            mimeType = mimeType.replace("resource", "application");
+        }
+
+        if(!mimeType.startsWith("application")) {
+            return false;
+        }
+
+        if(mimeType.endsWith("/folder")) {
+            return false;
+        }
+
+        return mimeType.endsWith("/zip");
```

Comment:
Perhaps support more archive types other than `zip`?

Reply:
I won't add/remove funcitonality in this PR.